keras_nlp/models/albert/albert_tokenizer.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, proto, **kwargs):
         cls_token = "[CLS]"
         sep_token = "[SEP]"
         pad_token = "<pad>"
+        mask_token = "<mask>"
         for token in [cls_token, sep_token, pad_token]:
```

Comment:
Oh sorry missed one thing! We should update our tokenizer to check for the masked token here. Which will then mean we have to update a lot of testing for albert_tokenizer and albert_preprocessor so they actually include the mask token.

Reply:
This PR shows how to add the mask in when training a sentencepiece model. https://github.com/keras-team/keras-nlp/pull/732